scripts/mkcloud

```diff
@@ -1155,81 +1065,19 @@ function cloudupgrade_reboot_and_redeploy_clients()
-    echo "   securitytests: install and run security test suite"
-    echo "   qa_test:       run the qa test suite"
-    echo "   help:          this usage"
-    echo
-    echo " Environment variables (need to be exported):"
-    echo
-    echo " Mandatory"
-    echo "   cloudpv=/dev/vdx (default /dev/vdb)"
-    echo "       :  LVM will be created on this device (at least 80GB)"
-    echo "   cloudsource=develcloud3 | develcloud4 | develcloud5 | susecloud5 | GM3 | GM4 | GM3+up | GM4+up | M?  (default '')"
-    echo "       : defines the installation source"
-    echo
-    echo " Optional"
-    echo "   hacloud='' | 1  (default='')"
-    echo "       : setup a high availability cloud, requires at least 7 (better 9) nodes"
-    echo "   upgrade_cloudsource='' (default='')"
-    echo "       : set new cloudsource for upgrade process"
-    echo "   TESTHEAD='' | 1  (default='')"
-    echo "       : use Media from Devel:Cloud:Staging and add test update repositories (except for GM* targets)"
-    echo "   cephvolumenumber  (default=0)"
-    echo "       : the number of 11GB ceph volumes that will be created per node"
-    echo "         note: proposal step does NOT contain a ceph proposal, do it manually"
-    echo "   nodenumber=2    (default 2)"
-    echo "       : set the number of nodes to be created (excl. admin node)"
-    echo "   nodenumberlonelynode=2    (default 0)"
-    echo "       : set the number of lonely nodes to be created in the admin network"
-    echo '   nodenumbercompute=1  (default $nodenumber resp. 1 in case of hacloud=1)'
-    echo "       :  set the number of compute nodes in a HA setup, the remaining nodes"
-    echo '          of $nodenumber - $nodenumercompute are used as HA cluster nodes'
-    echo "   vcpus=1         (default 1)"
-    echo "       : set the number of CPU cores per compute node"
-    echo "   adminvcpus=1    (default $vcpus)"
-    echo "       : set the number of CPU cores for admin node"
-    echo "   admin_node_memory (default 2097152)"
-    echo "       : Set the memory in KB assigned to admin node"
-    echo "   controller_node_memory (default 4194304)"
-    echo "       : Set the memory in KB assigned to compute nodes"
-    echo "   compute_node_memory (default 2097152)"
-    echo "       : Set the memory in KB assigned to compute nodes"
-    echo "   networkingplugin"
-    echo "       : Set the networking plugin to be used by neutron (e.g. openvswitch),"
-    echo "         if it isn't defined the barclamp-neutron's default is used."
-    echo "   networkingmode"
-    echo "       : Set the networking mode to be used by neutron (e.g. gre)"
-    echo "         if it isn't defined the barclamp-neutron's default is used."
-    echo
+
+    while read line
+    do
+        [[ $line =~ "\`mkcloud\` consists of" ]] && out=1
+        [[ $line =~ "\`\`\`" ]] && continue
+        [[ $line =~ "### Additional Information" ]] && exit 1
+        [[ $out == 1 ]] && echo $line
+    done < $docfile
```

Comment:
Maybe add a complain with a nice message that the usage is relying on a different file being present if the $docfile does not exist. This does not work when running mkcloud from a place where it was copied manually.